src/NovelRT/CMakeLists.txt

```diff
@@ -157,6 +157,7 @@ target_link_libraries(Engine
     png_static
     SndFile::SndFile
     spdlog
+    stduuid
```

Comment:
If `stduuid` is a header only library, we probably don't want to be doing this as it's assuming we're linking to the `stduuid` target and will require an export target to be installed.

Reply:
Thanks, fixed!